supervisor/supervisor.go

```diff
@@ -51,21 +51,51 @@ func New(stateDir, eventLogDir string, f factory.Factory, defaultCpus int, defau
 	return sv, sv.Events.setupEventLog(eventLogDir)
 }
 
-func (sv *Supervisor) CreateContainer(container, bundlePath, stdin, stdout, stderr string, spec *specs.Spec) (*Container, *Process, error) {
+func (sv *Supervisor) CreateContainer(container, bundlePath string, spec *specs.Spec) (c *Container, err error) {
```

Comment:
As `create` command takes all the flags of original start command, I think we should save the flag configs when run the `create` command, then use `start` command to do real process starting and IO handling.

Also, I think everytime we `create` and container, `runv-containerd` should listen on a socket named with container name such as `/run/runv/containers/myContainer/containerd.sock`, so that `start` command can search the `/run/runv/containers/` dir and find the containerd socket for some container.